docs/user-interface/images/app-icons.md

```diff
@@ -1,83 +1,215 @@
+
+The base size is specified with the `BaseSize="W,H"` attribute, where `W` is the width of the icon and `H` is the height of the icon. The value specified as the base size must be divisible by 8. The following example sets the base size:
 
 ```xml
-<MauiIcon Include="Resources\AppIcon\appicon.svg" Resize="false" />
+<ItemGroup>
+    <MauiIcon Include="Resources\AppIcon\appicon.png" BaseSize="128,128" />
+</ItemGroup>
 ```
 
-To add a tint to your app icon, which is useful when you have an icon or simple image you'd like to render in a different color to the source, set the `TintColor` attribute:
+And the following example stops the automatic resizing of a vector-based image:
 
 ```xml
-<MauiIcon Include="Resources\AppIcon\appicon.svg" TintColor="#66B3FF" />
+<ItemGroup>
+    <MauiIcon Include="Resources\AppIcon\appicon.svg" Resize="false" />
+</ItemGroup>
 ```
 
-A background color for the app icon can also be specified:
+## Recolor the background
+
+If the background image used in composing the app icon uses transparency, it can be recolored by specifying the `Color` attribute on the `<MauiIcon>`. The following example sets the background color of the app icon to red:
 
 ```xml
-<MauiIcon Include="Resources\AppIcon\appicon.svg" Color="#512BD4" />
+<ItemGroup>
+    <MauiIcon Include="Resources\AppIcon\appicon.svg" Color="#FF0000" />
+</ItemGroup>
 ```
 
-<!-- Valid color values are actually derived from the SKColor struct, rather than Microsoft.Maui.Graphics.Colors. -->
-Color values can be specified in hexadecimal, or as a .NET MAUI color. For example, `Color="Red"` is valid.
+Color values can be specified in hexadecimal, using the format: `#RRGGBB` or `#AARRGGBB`. The value of `RR` represents the red channel, `GG` the green channel, `BB` the blue channel, and `AA` the alpha channel. Instead of a hexadecimal value, you may use a named .NET MAUI color, such as `Red` or `PaleVioletRed`.
+
+## Recolor the foreground
 
-At build time, the app icon can be resized to the correct resolutions for the target platform and device. The resulting app icon is then added to your app package.
+If the app icon is composed of a background (`Include`) image and a foreground (`ForegroundFile`) image, the foreground image can be tinted. To tint the foreground image, specify a color with the `TintColor` attribute. The following example tints the foreground image yellow:
+
+```xml
+<ItemGroup>
+    <MauiIcon Include="Resources\AppIcon\appicon.png" ForegroundFile="Resources\AppIcon\appiconfg.svg" TintColor="Yellow" />
+</ItemGroup>
+```
+
+Color values can be specified in hexadecimal, using the format: `#RRGGBB` or `#AARRGGBB`. The value of `RR` represents the red channel, `GG` the green channel, `BB` the blue channel, and `AA` the alpha channel. Instead of a hexadecimal value, you may use a named .NET MAUI color, such as `Red` or `PaleVioletRed`. The named colors are from the <xref:SkiaSharp.SKColors?displayProperty=fullName> type.
```

Comment:
Re: you may use a named .NET MAUI color, such as `Red` or `PaleVioletRed`. The named colors are from the <xref:SkiaSharp.SKColors?displayProperty=fullName> type.

The two sentences are contradictory. .NET MAUI named colors are in Microsoft.Maui.Graphics.Colors. Then the second sentence says they are in SkiaSharp.SKColors. Microsoft.Maui.Graphics.Colors and SkiaSharp.SKColors should have identical named colors now. While it's technically accurate to say that the named colors come from SkiaSharp.SKColors, we don't want users to have to poke around SkiaSharp to understand what the named color values are (users will gain familiarity with MAUI's named color values from intellisense when setting Color properties on controls). Hence originally saying that the named colors come from .NET MAUI.

Reply:
Oops, I meant to take out the `.NET MAUI named color`, that was from the commented out text. However, with what you clarified about the color naming, I think I'll take out the SKColor link.